### PR TITLE
User-facing strings

### PR DESCRIPTION
**Pull request by indirectlylit**
Updates to default user-facing strings. Attempting to conform to [Material writing guidelines](https://material.io/guidelines/style/writing.html)

### CODE REVIEW

File: kolibri/plugins/management/assets/src/vue/manage-content-page/task-status.vue [hunk 1]
```diff
@@ -26,13 +26,13 @@
     $trs: {
       buttonClose: 'Close',
       buttonCancel: 'Cancel',
-      failed: 'Failed.',
-      failed_msg: 'The transfer did not succeed. If you restart, any progress will be retained.',
+      failed: 'Try again',
```
Comment (radinamatic) [inline kolibri/plugins/management/assets/src/vue/manage-content-page/task-status.vue]: Maybe `Please, try again`?

File: kolibri/core/assets/src/vue/error-box/index.vue [hunk 2]
```diff
@@ -21,9 +21,9 @@
     $trNameSpace: 'error',
     $trs: {
       errorHeader: 'Error',
-      errorButtonLabel: 'Hide Error',
+      errorButtonLabel: 'Hide error',
       // eslint-disable-next-line
-      explanation: `Sorry, we've encountered an issue. You may need to refresh the page or click one of the main navigation links.`,
+      explanation: `Unknown issue. You may need to refresh`,
```
Comment (radinamatic) [inline kolibri/core/assets/src/vue/error-box/index.vue]: I really, really dislike "Unknown issue" messages... Isn't there any recommended action we can suggest to the user to escalate the issue? 

How does this sound?
`Ups, something went wrong! Please try refreshing the page and if the problem persists, contact your administrator.`

Comment (indirectlylit) [reply kolibri/core/assets/src/vue/error-box/index.vue]: Fair enough. I was trying to follow the tone guide: [tone guidelines](https://material.io/guidelines/style/writing.html#writing-tone)

Not all users will be able to contact an administrator. Maybe something simple like this?

> Sorry, something went wrong. Please try refreshing the page

Comment (radinamatic) [reply kolibri/core/assets/src/vue/error-box/index.vue]: Yep, I believe my suggestion was actually in line with the first guideline.
Good point about the administrator!

**Pull request merged**